sw/airborne/modules/computer_vision/bebop_ae_awb.c

```diff
@@ -352,5 +352,5 @@ void bebop_ae_awb_init(void)
   awb_set.gain_scheduling_tolerance = BEBOP_AWB_GAIN_SCHEDULING_TOLERANCE;
   awb_set.gain_scheduling_step      = BEBOP_AWB_GAIN_SCHEDULING_STEP;
 
-  cv_add_to_device_async(&front_camera, update_ae_awb, BEBOP_AE_AWB_NICE, 0);
+  cv_add_to_device_async(&front_camera, update_ae_awb, BEBOP_AE_AWB_NICE, 0, 0);
```

Comment:
why not using the `camera_id ` ? and if not using it, it should be marked as unused attribute to prevent warning

Reply:
Do you mean the `update_ae_awb` function not using the camera_id or in the line where `cv_add_to_device_async` is called? 

If it's the former, I can change the function signature to `static struct image_t *update_ae_awb(struct image_t *img, uint8_t __attribute__((unused)) camera_id)` to make it clearer that the field is only there because of how `cv_run_device` behaves (it now needs to pass an additional field on top of img, camera_id, to the callback of `cv_function`). 

In that case, I will then also change all function signatures where I had added this parameter originally but not used it.